Source/Core/ServiceTask.swift

```diff
@@ -107,6 +107,12 @@ import Foundation
 
 extension ServiceTask {
     /// TODO: Needs docs
+    @discardableResult public func shouldHandleCookies(_ handle: Bool) -> Self {
+        urlRequest.httpShouldHandleCookies = handle
```

Comment:
We cannot directly mutate `urlRequest` since it's value will be overwritten when the `Request` value is converted to a `URLRequest` here: https://github.com/Electrode-iOS/ELWebService/blob/master/Source/Core/ServiceTask.swift#L40

Instead, please add a `shouldHandleCookies` property to `Request` so the `httpShouldHandleCookies` property is set during the [`URLRequest` encoding process](https://github.com/Electrode-iOS/ELWebService/blob/master/Source/Core/Request.swift#L209-L247). Then you can use the `ServiceTask` API to set the new `Request` property.